Added a sleep when polling OSD Jobs

bull-simon-hill opened the pull request:
Added a sleep to work around issue:
https://github.com/dell/dellemc-openmanage-ansible-modules/issues/32

This appears to be a timing issue.  Adding a 10 second sleep allows the ansible module to reliably get the Job status return without failing.

I am not sure this is best place for this sleep but does seem like a simple work around to allow it to function correctly until this could be properly fixed.

bull-simon-hill committed:
Added a sleep when polling OSD Jobs

Added a sleep to work around issue:
https://github.com/dell/dellemc-openmanage-ansible-modules/issues/32

digifuchsi commented:
Fix is working for me.
If Dell hasn't any better solution this should be merged.

rajeevarakkal commented on omdrivers/lifecycle/iDRAC/iDRACJobs.py:
```diff
@@ -147,6 +147,7 @@ def get_job_status(self, jobid):
         jobret = {"Status": TypeHelper.resolve(JobStatusEnum.InProgress)}
         if jobid.startswith('DCIM_OSD'):
             # Poll for OSD Concrete Job
+            time.sleep( 10 )
```
```suggestion
```

you can delete this sleep time and add it in the calling section

Add a  `time.sleep( 30 )` wait time on line # 242 immediately after the while loop starting. And also better to remove  `time.sleep( 5 )` at line #275. Over all this will help to have sleep interval for job polling at a single place rather than at two places. Otherwise this changes looks good to accept.

bull-simon-hill replied:
I have updated as you suggested.  Thanks @rajeevarakkal

bull-simon-hill committed:
Update iDRACJobs.py

Updated as per this discussion with @rajeevarakkal
https://github.com/dell/omsdk/pull/8#discussion_r319363688
Sleep moved to calling code and sleep at the end of the loop removed.

rajeevarakkal commented:
@simonhill-atos; sure, we will merge it

sqrlmstr5000 commented:
When do we expect this to make it to the master branch and a prod release of omsdk?
